### DISCUSSION TITLE
对于【首字下沉】会出现漏检的情况，请问这是需要对检测模型进行参数更改吗？为了方便就用了WEB DEMO的结果作为说明

### DISCUSSION THREAD

**Discussion by Thunderltx** [PaddlePaddle/PaddleOCR#12698] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：WEB DEMO
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): YES
![微信截图_20231213132753](https://github.com/PaddlePaddle/PaddleOCR/assets/37621028/ccdb0789-5576-440d-98e0-194758c3674c)

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by Thunderltx** [comment]
段落中有个别字符的字号和其他字符不一样，对于这个【个别】的字符，检测就会出现问题
![微信截图_20231213133014](https://github.com/PaddlePaddle/PaddleOCR/assets/37621028/c988c0b7-d23d-4992-81d8-af52dcfdc6a0)

**Reply by SWHL** [reply]
检测模型训练时，就没有用到这类数据。所以效果自然差些。
这种需要构建数据集微调了。而且这涉及到后续文本识别模型，文本识别模型中也没有训练这种数据，识别效果也会差

**Comment by YoLi-sw** [comment]
mark
